src/main/java/org/mamute/model/Tag.java

```diff
@@ -48,7 +54,8 @@ public Tag() {
 	
 	public Tag(String name, String description, User author) {
 		this.name = name.toLowerCase();
-		this.description = description;
+        this.sluggedName = toSlug(name);
+        this.description = description;
```

Comment:
Of course. Should this project maybe have a wiki with coding standards and editor settings explained? I imagine most of them are industry standard in Java world, but not everybody are from Java world (khm, people like me:)